src/ring_theory/unique_factorization_domain.lean

```diff
@@ -1190,6 +1195,13 @@ begin
     exact (zero_lt_one.trans_le h).ne' }
 end
 
+theorem count_self [nontrivial α] {p : associates α} (hp : irreducible p) :
+  p.count p.factors = 1 :=
+begin
+  rw [factors_self hp, associates.count_some hp],
+  exact multiset.count_singleton_self _,
+end
```

Comment:
```suggestion
by simp [factors_self hp, associates.count_some hp, multiset.count_singleton_self]
```
I don't understand why `multiset.count_singleton_self` is not a `simp` lemma. Can you try to make it `simp` and see if something breaks? If not, you can remove it in this line.

Reply:
I made it `simp` and nothing broke on that file. If something breaks when building mathlib I will change it back. Thanks!